middlewares/hacker.middleware.js

```diff
@@ -241,6 +241,40 @@ async function updateStatusIfApplicationCompleted(req, res, next) {
     }
 }
 
+/**
+ * Checks that the hacker's status is appropriate to be checked in by a volunteer.
+ * The status should be 'Confirmed'
+ * @param {{params:{id: string}, body: *}} req 
+ * @param {*} res 
+ * @param {*} next 
+ */
+async function validateCheckInHackerStatus(req, res, next) {
+    const hacker = await Services.Hacker.findById(req.params.id, req.body);
+    if (hacker) {
+        const status = hacker.status;
+
+        if (status !== Constants.General.HACKER_STATUS_CONFIRMED) {
```

Comment:
@krubenok should we allow people to check in if they have not been confirmed?

Reply:
In other words, can you define how you want the process for walk-ins to work?

Reply:
Talked to @krubenok, it should be confirmed and accepted.